samples/bot-conversation/python/infra/azure.bicep

```diff
@@ -26,7 +26,7 @@ resource botService 'Microsoft.BotService/botServices@2021-03-01' = {
     endpoint: 'https://${botAppDomain}/api/messages'
     msaAppId: botAadAppClientId
     msaAppType: microsoftAppType
```

Comment:
If Python botbuilder only supports multi tenant app, we can hard code here to reduce the environment variable count

Reply:
Can also remove the script action that generates the environment variables in teamsapp.local.yml

Reply:
done. also removed properties in azure.parameters.json